qiskit/circuit/library/data_preparation/pauli_feature_map.py

```diff
@@ -99,8 +104,9 @@ class PauliFeatureMap(NLocal):
         OrderedDict([('cx', 39), ('rx', 36), ('u1', 21), ('h', 15), ('ry', 12), ('rz', 12)])
 
     References:
-        [1]: Havlicek et al. (2018), Supervised learning with quantum enhanced feature spaces.
-           `arXiv:1804.11326 <https://arxiv.org/abs/1804.11326>`_
+        [1] Havlicek et al. Supervised learning with quantum enhanced feature spaces.
```

Comment:
Something looks add in the reference section:

<img width="768" alt="Screenshot 2023-05-06 at 11 15 12" src="https://user-images.githubusercontent.com/766693/236615192-65fd67e7-b6d3-4aab-9b76-372d101e4c32.png">

Reply:
Thanks for checking the docs 😁 Should be better in 8d90474